.ddev/config.yaml

```diff
@@ -0,0 +1,174 @@
+name: tea
+type: php
+docroot: ""
+php_version: "7.4"
+webserver_type: nginx-fpm
+router_http_port: "80"
+router_https_port: "443"
+xdebug_enabled: true
+additional_hostnames: []
+additional_fqdns: []
+mariadb_version: "10.5"
+mysql_version: ""
+provider: default
+hooks:
+  post-start:
+  - exec: composer install --no-progress
+use_dns_when_possible: true
+composer_version: ""
+
+
+# This config.yaml was created with ddev version v1.16.7
```

Comment:
And today, ddev 1.17.0 has been released. So we might want to re-generate the file again:

https://github.com/drud/ddev/releases/tag/v1.17.0